patches/server/0126-Multithreaded-Tracker.patch

```diff
@@ -435,7 +435,9 @@ index 0000000000000000000000000000000000000000..1ecaa9f4b4843c05af2139ae3e4b0dd0
 +                tickAsyncWithCompatMode(level);
 +            }
 +        } catch (Exception e) {
-+            LOGGER.error("Failed to execute async task.", e);
++            LOGGER.error("Error occurred while executing async task.");
++            //noinspection CallToPrintStackTrace
++            e.printStackTrace();
```

Comment:
Original method can also print stacktrace. Is there a reason to use printStackTrace() ?

Reply:
Done. uwu